clang-tools-extra/test/clang-doc/single-source-html.cpp

```diff
@@ -1,2 +1,2 @@
 // RUN: clang-doc --format=html --executor=standalone %s -output=%t/docs | FileCheck %s
-// CHECK: Using default asset: {{.*}}..\share\clang
\ No newline at end of file
+// CHECK: Using default asset: {{.*}}{{[\\/]}}share{{[\\/]}}clang
```

Comment:
https://releases.llvm.org/14.0.0/docs/TestingGuide.html

See ‘pathsep’

Reply:
I think pathsep is looks for : (or ; on Windows)